docs/faq.md

```diff
@@ -48,3 +48,23 @@ New password for default administrator (user-xxxxx):
 ### I added an additional disk with partitions. Why is it not getting detected?
 
 As of Harvester v1.0.2, we no longer support adding additional partitioned disks, so be sure to delete all partitions first (e.g., using `fdisk`).
+
+### Why are there some Harvester pods that become ErrImagePull/ImagePullBackOff?
+
+This is likely because your Harvester cluster is an air-gapped setup, and some pre-loaded container images are missing. Kubernetes has a mechanism that does garbage collection against bloated image stores. When the partition which stores container images is over 85% full, `kubelet` will try to prune some least used images to save disk space until the occupancy is lower than 80%. These numbers (85% and 80%) are default High/Low thresholds that come with Kubernetes.
+
+To recover from the state, either pull the missing images from the Internet (by setting up an HTTP proxy if it's an air-gapped environment) or manually import the images from the ISO image. For multi-node clusters, chances are the missing container images on one node could be found on the other nodes. If that's the case, you can export the images from the node that still has them and import them on the missing one.
```

Comment:
```suggestion
To recover from this state for a single node cluster, do one of the following:  
    - If it's an air-gapped environment, pull the missing images from the Internet by setting up an HTTP proxy.
    - Manually import the images from the ISO image. 
    
For multi-node clusters, you can likely find the missing container images on one node on the other nodes. If that's the case, you can export the images from the node still with them and import them on the missing one.
```

Reply:
The two bullet points are not applied to single-node cases only. And "finding missing container images on the other nodes" implies it's a multi-node case. So I'd like to combine your suggestions and rewrite it like the following:
```suggestion
To recover from this state, do one of the following depending on the cluster's configuration:
- Pull the missing images from sources outside of the cluster (if it's an air-gapped environment, you might need to set up an HTTP proxy beforehand)
- Manually import the images from the Harvester ISO image
- Find the missing images on one node on the other nodes, and then export the images from the node still with them and import them on the missing one
```
Thank you!